testsuite/features/build_validation/add_custom_repositories/add_rocky9_repositories.feature

```diff
@@ -88,4 +92,23 @@ Feature: Add the Rocky 9 distribution custom repositories
     And I click on "Create"
     Then I should see a "Content Lifecycle Project - Remove AppStream metadata" text
     When I click on "Attach/Detach Sources"
-    # TODO: the EL9 products are not ready yet
+    And I select "Rocky Linux 9" from "selectedBaseChannel"
+    And I check "Custom Channel for Rocky 9 DVD"
+    And I check "Rocky Linux 9 AppStream"
+    And I click on "Save"
+    Then I should see a "Custom Channel for Rocky 9 DVD" text
+    When I click on "Attach/Detach Filters"
+    And I check "python-3.6: enable module python36:3.6"
```

Comment:
I think this and the next one are not needed for EL9, according to the doc (check it again), and the tests I did for Alma/Oracle

And in fact, they don't exist on RHEL9, according to https://access.redhat.com/support/policy/updates/rhel-app-streams-life-cycle (so it should be the same for the clones)

Reply:
As discussed at Slack, if this is just to test that filters are working, better use something that exists, FMPOV :-)

Reply:
right, fixed, thanks.